<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NES Space Checker Qt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>NESsc_Icon.ico</normaloff>
    <disabledoff>NESsc_Icon.ico</disabledoff>NESsc_Icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="actionWatch"/>
    <addaction name="separator"/>
    <addaction name="actionSave_report"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEmpty_value">
    <property name="title">
     <string>Empty value</string>
    </property>
    <addaction name="actionAuto"/>
    <addaction name="action0x00"/>
    <addaction name="action0xff"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShiru_s_style"/>
    <addaction name="actionChart"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmpty_value"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionWatch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Watch</string>
   </property>
  </action>
  <action name="actionSave_report">
   <property name="text">
    <string>Save report</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="action0x00">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0x00</string>
   </property>
  </action>
  <action name="action0xff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0xff</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShiru_s_style">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shiru's style</string>
   </property>
  </action>
  <action name="actionChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chart</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
